### ISSUE TITLE
Code should not be hyphenated

### ISSUE THREAD

**Issue by lebigot** [sphinx-doc/sphinx#2917]
Code like `umath.atan(umath.tan(x))` (with a double backquote on each side) can get hyphenated as `umath.atan(umath.-tan(x))`, which is meaningless and confusing. Example: https://pythonhosted.org/uncertainties/tech_guide.html#constraints-on-the-uncertainties; changing the window width demonstrates the problem.

(Sphinx version 1.4.6.)

**Comment by tk0miya** [comment]
Really? I can't reproduce with Chrome, firefox and opera.
These browsers basically folds before or after piece of codes:
![2016-08-29 0 27 18](https://cloud.githubusercontent.com/assets/748828/18034768/c9ced098-6d80-11e6-913d-9e4953bdc5a9.png)

Could you tell me your situation in detail?

**Comment by lebigot** [comment]
Ah: this is with Safari 9.1.2 on OS X 10.11.6.

<img width="826" alt="bad_hyphenation" src="https://cloud.githubusercontent.com/assets/5793/18034806/c09cf354-6d46-11e6-953f-db7445739cef.png">

**Comment by tk0miya** [comment]
I tried with Safari-9.1.1 on OS X 10.11.5 again. But it folded before `umath.atan` too:
![2016-08-29 0 49 11](https://cloud.githubusercontent.com/assets/748828/18034867/fe3415c6-6d82-11e6-9668-0cd764c30f08.png)

Surely, Sphinx defines hyphenation styles. But it only works when `language` is set in `conf.py`. But your document seems the `language` is not set (if set, `lang` attribute will be appeared in `<html>` tag).

Anyway, current styles is not good as you said. I will take a look later.

**Comment by lebigot** [comment]
Thank you for looking into this.

Hmm, this is strange: words are definitely hyphenated, in Safari… I disabled all Safari extensions, just to see, but this did not change anything.

They are not hyphenated, with Chrome…

**Comment by tk0miya** [comment]
Oh sorry. Chrome does not support hyphenation styles:
ref: https://developer.mozilla.org/en-US/docs/Web/CSS/hyphens

**Comment by tk0miya** [comment]
I reproduced with folloing command:

```
$ git clone https://github.com/lebigot/uncertainties/
$ cd uncertainties/doc
$ echo 'language = "en"' >> conf.py
$ make html
$ open -a Safari _build/html/tech_guide.html
```

Adding `language` setting is a key point.

**Comment by tk0miya** [comment]
Fixed at 37bb9c2.
Thank you for reporting :-)

**Comment by lebigot** [comment]
Thank you for fixing this quickly!
